test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -55,14 +60,10 @@ public class FightRequestTest {
 
   @BeforeEach
   public void beforeEach() {
-    GenericRequest.passwordHash = "";
-    Preferences.saveSettingsToFile = false;
```

Comment:
Why? Doesn't this leave a Preferences file in test/root or something?

That seems like opposite of "cleaning up after itself" since that file will carry over to other tests with the same user name, no?

Reply:
We don't do this everywhere so I was removing for consistency - we should really make a very light base class for tests that does this and a logout/login